### PR TITLE
Manual Scaling from cli

### PR DESCRIPTION
**Pull request by felipevicens**

### CODE REVIEW

File: src/tnglib/requests.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
 import os
 import yaml
 import tnglib.env as env
+import tnglib.functions as func
```
Comment (tsoenen) [inline src/tnglib/requests.py]: I think this can be removed,no?

File: src/tngcli/cli.py [hunk 2]
```diff
@@ -684,6 +692,54 @@ def parse_args(args):
...
+                            default=False,
+                            help='Specify the vnf descriptor uuid')
+
+    parser_req.add_argument('--vnf_uuid',
+                            metavar='vnfd_uuid',
+                            required=False,
+                            default=False,
+                            help='Specify the vnf instance uuid')
+
+    parser_req.add_argument('--service_uuid',
```
Comment (tsoenen) [inline src/tngcli/cli.py]: What is this for? Isn't it the same as instance_uuid?

File: src/tnglib/requests.py (hunk 1 above)
Comment (felipevicens) [reply src/tnglib/requests.py]: Yes! removing it

File: src/tngcli/cli.py (hunk 2 above)
Comment (felipevicens) [reply src/tngcli/cli.py]: Yes, the instantiation if from services, not from here. Removed

**Pull request merged**